### DISCUSSION TITLE
Having problem to get recent activity

### DISCUSSION THREAD

**Discussion by sirrik** [cwendt94/espn-api#199] [ESPN API] [answered]
I successfully integrated with my public league(Basketball). I tried a few things and all worked well but when it comes to "league.recent_activity()" with or without any parameters returns Exception("Access Denied"). As I can see it's the 401 Unauthorized issue. So, I checked the discussions and could not found anything about it. Any ideas why?

Amazing project by the way!! Thanks.

**Comment by cwendt94** [comment, marked as answer]
Recent activity is one of the functions that require authentication even if your league is public. So if I try to go to your league on ESPN's website and look for recent activity I will get Access Denied.

I have updated the wiki docs under recent activity section: https://github.com/cwendt94/espn-api/wiki/League-Class-Basketball

**Reply by sirrik** [reply]
Okay, I will try with espn_s2 and swid. Thanks for the quick response.
